### PR TITLE
New version: wethat.onenotetaggingkit version 4.0.8132

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57124)

### CODE REVIEW

File: manifests/w/wethat/onenotetaggingkit/4.0.8132/wethat.onenotetaggingkit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: wethat.onenotetaggingkit
+PackageVersion: 4.0.8132
```
Comment (OfficialEsco) [inline manifests/w/wethat/onenotetaggingkit/4.0.8132/wethat.onenotetaggingkit.installer.yaml:5]:
```suggestion
PackageVersion: 4.0.8132.30888
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/57099

File: manifests/w/wethat/onenotetaggingkit/4.0.8132/wethat.onenotetaggingkit.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: wethat.onenotetaggingkit
+PackageVersion: 4.0.8132
```
Comment (ItzLevvie) [inline manifests/w/wethat/onenotetaggingkit/4.0.8132/wethat.onenotetaggingkit.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.0.8132.30888
```

File: manifests/w/wethat/onenotetaggingkit/4.0.8132/wethat.onenotetaggingkit.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: wethat.onenotetaggingkit
+PackageVersion: 4.0.8132
```
Comment (ItzLevvie) [inline manifests/w/wethat/onenotetaggingkit/4.0.8132/wethat.onenotetaggingkit.yaml:5]:
```suggestion
PackageVersion: 4.0.8132.30888
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**